ext/zend_test/test.stub.php

```diff
@@ -17,6 +17,7 @@ class _ZendTestClass implements _ZendTestInterface {
         public int $intProp = 123;
         public ?stdClass $classProp = null;
         public stdClass|Iterator|null $classUnionProp = null;
+        public readonly int $readonlyProp;
```

Comment:
I'm a bit puzzled by this: if I run gen_stub.php with my default PHP version (8.0), I get a parser error because of `readonly`. When do stub files get parsed by PHP? 🤔  I thought that only PHP-Parser parses them. Or is there any issue with token emulation?

Reply:
You need to switch to `Lexer\Emulative` in https://github.com/php/php-src/blob/f3a662cf3cbfba0e659a5962554951dee698e76b/build/gen_stub.php#L1813.